libs/sdk-core/src/reverseswap.rs

```diff
@@ -367,18 +367,31 @@ impl BTCSendSwap {
         // Depending on the new state, decide next steps and transition to the new state
         for rs in monitored {
             debug!("Checking monitored reverse swap {rs:?}");
-            // (Re-)Broadcast the claim tx for monitored reverse swaps that have a confirmed lockup tx
-            if matches!(self.get_lockup_tx_status(&rs).await?, TxStatus::Confirmed) {
-                info!("Lock tx is confirmed, preparing claim tx");
-                let claim_tx = self.create_claim_tx(&rs).await?;
-                let claim_tx_broadcast_res = self
-                    .chain_service
-                    .broadcast_transaction(serialize(&claim_tx))
-                    .await;
-                match claim_tx_broadcast_res {
-                    Ok(txid) => info!("Claim tx was broadcast with txid {txid}"),
-                    Err(e) => error!("Claim tx failed to broadcast: {e}"),
-                }
+
+            let lockup_tx_status = self.get_lockup_tx_status(&rs).await?;
+            let claim_tx_status = self.get_claim_tx_status(&rs).await?;
+
+            match claim_tx_status {
```

Comment:
In a glance it looks correct but one thing is unclear to me: 
The  `execute_pending_reverse_swaps` is only called after `refresh_monitored_reverse_swaps` so the question is why the status wasn't updated to Completed? Shouldn't the provider return Completed here (https://github.com/breez/breez-sdk/blob/main/libs/sdk-core/src/reverseswap.rs#L475) in this case?